boolean/nand/a5f368aa-2cea-4110-a170-c08ec18a6a63.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "boolean/nand",
```

Comment:
Can you please start the name with a capital letter? like `boolean/Nand`
... again this is question of standards that we'll publish very soon now ...

Reply:
Capitalised as requeted :)